README.md

```diff
@@ -35,6 +38,12 @@ make deploy                     # install controller with kubectl
 kubectl delete pod/verify-metadata-controller-controller-manager-0
 ```
 
+To get the project to run and build you need to place it under your `$GOROOT` which is typically set to `~/go/src/` the path for the project should look something like `~/go/src/github.com/alphagov/verify-metadata-controller`.
+
+Once you have dep installed you should be able to run `dep ensure` from the root of the project.
```

Comment:
What does `dep` bring to the party?

Reply:
Dep is what is used by kubebuilder to provision the dependancies for Go when the project is initially created. Its just re running the install of the dependancies for a project thats already been created.